test/distributed/rpc/process_group/process_group_rpc_agent_mixin.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+
+class ProcessGroupRpcAgentMixin(object):
+    @property
+    def rpc_backend_name(self):
```

Comment:
will there be any other helper function be added to this class? If this is the only one, this looks like an overkill for just passing a string to a test class.

Reply:
Yes, it's `def rpc_agent_options` in the next PR. Adding this Mixin + refactoring for this Mixin is the purpose of this PR.